libraries/AdaptiveExpressions/Expression.cs

```diff
@@ -113,158 +113,9 @@ public Expression(ExpressionEvaluator evaluator, params Expression[] children)
         /// <returns>resulting expression.</returns>
         public static Expression Parse(string expression)
         {
-            return new ExpressionEngine().Parse(expression);
+            return new ExpressionParser().Parse(expression);
```

Comment:
why not moving this also into expression factory

Reply:
NO, I think it is fewer concept count to have it be on expression. 
Expression.And(Expression.Lamda(() => ...), Expression.Parse("user.age > 3"))
is WAY better then
ExpressionBuilder.And(ExpressionBuilder.Lamda(() => ...), ExpressionBuilder.Parse("user.age > 3))

Only one class, and it's all about Expressions and discoverable off of expression.


---
In reply to: [386273952](https://github.com/microsoft/botbuilder-dotnet/pull/3471#discussion_r386273952) [](ancestors = 386273952)